opensrp-chw/src/main/java/org/smartregister/chw/interactor/FpUpcomingServicesInteractor.java

```diff
@@ -0,0 +1,6 @@
+package org.smartregister.chw.interactor;
+
+import org.smartregister.chw.core.interactor.CoreFpUpcomingServicesInteractor;
+
+public class FpUpcomingServicesInteractor extends CoreFpUpcomingServicesInteractor {
```

Comment:
Since there are no custom implementations of the extended class in this class,  this file should be deleted. 
It isn't being used/referenced anywhere in the code.

Reply:
Deleted